imports/ui/clients/searchClient.html

```diff
@@ -79,4 +79,9 @@ <h4 class="text-center">Pending Clients</h4>
       DOB: {{formatDate dob}}
     </div>
   {{/if}}
+  {{#if viewAll}}
```

Comment:
add `View more` button at the bottom of the list (in the footer), not for every item